<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminCreerConcoursFrame</class>
 <widget class="QFrame" name="AdminCreerConcoursFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Ajout d'un concours&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date limite :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de resultat : </string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="champ_date_limite">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>488</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="champ_date_resultat">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>488</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enregistrer">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>660</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>411</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ann√©e :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="champ_annee"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>411</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Age limite :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="champ_age_limite"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
